### ISSUE TITLE
Hot Reload is sluggish since v0.30

### ISSUE THREAD

**Issue by jan-wilhelm** [blitz-js/blitz#1970]
**What is the problem?**

Since upgrading to one of the 0.30 canaries (can't pinpoint which one it was), the Hot Module Reload has been slower than before. It now consistently takes around 3-5 seconds before the NextJS reloading indicator even shows up, compared to a basically instantaneous HMR with pure NextJS.

@engelkes-finstreet [pointed out](https://discord.com/channels/802917734999523368/802989232582688788/812304749067501579) that he saw the same issue on the Discord.

**What are detailed steps to reproduce this?**

1. Run a Blitz application with BlitzJS >= 0.30.canary* or 0.30.1 in development mode
2. Change any code
3. Switch back to the browser window and wait for the NextJS loading indicator to show up in the bottom right. This takes a few seconds for me, and changes to my code show up only after about 5 seconds on my machine.

**Run `blitz -v` and paste the output here:**

```
macOS Big Sur | darwin-x64 | Node: v15.6.0

blitz: 0.30.1 (global)
blitz: 0.30.1 (local)

  Package manager: yarn 
  System:
    OS: macOS 11.2.1
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 64.02 MB / 16.00 GB
    Shell: 5.8 - /usr/local/bin/zsh
  Binaries:
    Node: 15.6.0 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 7.4.3 - /usr/local/bin/npm
    Watchman: Not Found
  npmPackages:
    @prisma/client: 2.17.0 => 2.17.0 
    blitz: 0.30.1 => 0.30.1 
    prisma: 2.17.0 => 2.17.0 
    react: 0.0.0-experimental-3310209d0 => 0.0.0-experimental-3310209d0 
    react-dom: 0.0.0-experimental-3310209d0 => 0.0.0-experimental-3310209d0 
    typescript: 4.1.5 => 4.1.5 
```

**Comment by flybayer** [comment]
This should be fixed in `0.30.1`. I changed some settings in that release. Can you double check to make sure you still have it in `0.30.1`?

**Comment by engelkes-finstreet** [comment]
For me it is fixed in 0.30.1. Back to normal speed 🚀

**Comment by jan-wilhelm** [comment]
yes, you're right. 0.30.1 is good again, and so is 0.30.2 :)
